connector_exchange/models/exchange_backend/common.py

```diff
@@ -170,15 +167,15 @@ def import_user_calendar(self):
                 # and if categories contains Odoo
                 for exchange_event in exchange_events:
                     sensitivity = exchange_event.sensitivity
-                    odoo_categ = False
-                    if not exchange_event.categories:
-                        continue
-                    for categ in exchange_event.categories:
-                        if categ == 'Odoo':
-                            odoo_categ = True
-                            break
-                    if odoo_categ and sensitivity not in \
-                            ['Private', 'Personal']:
+                    # odoo_categ = False
```

Comment:
I see this is commented. If you agree we can remove this completely and not to make any distinctions in category.

Reply:
:+1: I just "removed" to keep this under my eyes in case the code did not worked after :smile:

Reply:
can we merge this one and I will clean all this once your PR to our repo is merged?

Reply:
I would like to avoid making PR on PR with several forks